substrate/frame/support/procedural/src/construct_runtime/expand/metadata.rs

```diff
@@ -119,16 +119,16 @@ pub fn expand_runtime_metadata(
 						call_ty,
 						signature_ty,
 						extra_ty,
-						signed_extensions: <
+						extensions: <
 								<
 									#extrinsic as #scrate::sp_runtime::traits::ExtrinsicMetadata
-								>::SignedExtensions as #scrate::sp_runtime::traits::SignedExtension
+								>::SignedExtensions as #scrate::sp_runtime::traits::TransactionExtensionBase
 							>::metadata()
 								.into_iter()
-								.map(|meta| #scrate::__private::metadata_ir::SignedExtensionMetadataIR {
+								.map(|meta| #scrate::__private::metadata_ir::TransactionExtensionMetadataIR {
```

Comment:
This seems like it will be a breaking change in the metadata as well? will a field be renamed, or is it the same?

Reply:
It's the intermediary representation of metadata, so it's not a breaking change.